docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -12,10 +12,10 @@ import { getComponentPathname, typeOrder, repoURL } from 'docs/src/utils'
 
 const pkg = require('../../../../package.json')
 const componentMenu = require('docs/src/componentMenu')
+const behaviorMenuItems = require('docs/src/componentMenuBehaviors')
```

Comment:
I would rather suggest to use `behaviorMenu` as a file name (and `behaviorMenu` as a variable name):

```js
const behaviorMenu = require('docs/src/behaviorMenu')
```

This will result in consistent pattern with the line above: 

```js
const componentMenu = require('docs/src/componentMenu')
```